### PR TITLE
Clarify how reported usage works for failed/canceled jobs

### PR DESCRIPTION
**Pull request by beckykd**

### CODE REVIEW

File: docs/guides/execution-modes.mdx [hunk 1]
```diff
@@ -52,3 +52,12 @@ To find the usage time for your session or batch, in `qiskit-ibm-runtime` 0.23 o
...
+
+<span id="failed-job"></span>
+## Failed and canceled jobs
+
+When a job is failed or canceled, you are charged for usage as follows:
+
+* Job or batch mode: you are charged for all the time the QPU was locked for the execution, regardless of cancellations and errors. So if the error occurs before the lock, you are charged nothing. If the error/cancellation occurs after the system lock, you are charged based on the value that  _channels service_ returns as `consumed`. Thus, some jobs failed jobs have no cost and others do.
```
Comment (abbycross) [inline docs/guides/execution-modes.mdx:61]:
```suggestion
* Job or batch mode: you are charged for all the time the QPU was locked for the execution, regardless of cancellations and errors. So if the error occurs before the lock, you are charged nothing. If the error/cancellation occurs after the system lock, you are charged based on the value that  _channels service_ returns as `consumed`. Thus, some failed jobs have no cost and others do.
```

Comment (abbycross) [inline docs/guides/execution-modes.mdx:61]: Will _channels service_ be understood by readers?

**Review by abbycross — APPROVED** [review]

**Comment by abbycross** [comment]
@beckykd what do you think about also adding it as a FAQ to the execution-modes-faq page?

File: docs/guides/execution-modes.mdx [hunk 2]
```diff
@@ -41,7 +41,7 @@ The above are general guidelines, and you should tune your workload to find the
 
 ## Sessions versus batch usage
 
-Usage is a measurement of the amount of time the QPU is locked for your workload.
+Usage is a measurement of the amount of time the QPU is locked for your workload. It is important to understand how usage is calculated because it impacts how much you are charged for jobs (including failed jobs).
```
Comment (abbycross) [inline docs/guides/execution-modes.mdx:44]:
```suggestion
Usage is a measurement of the amount of time the QPU is locked for your workload. Failed or canceled jobs count toward your reported usage in certain circumstances - see the [Failed and canceled jobs](#failed-job) section for more details.
```

File: docs/guides/execution-modes.mdx [hunk 3]
```diff
@@ -52,3 +52,12 @@ To find the usage time for your session or batch, in `qiskit-ibm-runtime` 0.23 o
...
+
+<span id="failed-job"></span>
+## Failed and canceled jobs
+
+When a job is failed or canceled, you are charged for usage as follows:
+
+* Job or batch mode: you are charged for all the time the QPU was locked for the execution, regardless of cancellations and errors. So if the error occurs before the lock, you are charged nothing. If the error/cancellation occurs after the system lock, you are charged based on the value that   Qiskit Runtime returns as `consumed`. Thus, some failed jobs have no cost and others do.
+
+* Session mode: You are charged all the wall-clock time from the first job execution until the session goes inactive or is closed, regardless of numbers of jobs, cancellations, or errors.
```
Comment (abbycross) [inline docs/guides/execution-modes.mdx:63]:
```suggestion
When a job is failed or canceled, your usage is reported as follows:

* Job or batch mode: usage is reported for all the time the QPU was locked for the execution, regardless of cancellations and errors. If the error occurs before the lock, no usage is reported. If the error/cancellation occurs after the system lock, usage is reported based on the value that Qiskit Runtime returns as `consumed`. Thus, some failed jobs do not appear in your reported usage and others do.

* Session mode: Usage is reported for all the wall-clock time from the first job execution until the session goes inactive or is closed, regardless of numbers of jobs, cancellations, or errors.
```

**Review by javabster — APPROVED** [review]
LGTM 🚀  This is very helpful clarification to how usage is tracked!

File: docs/guides/fair-share-scheduler.mdx [hunk 4]
```diff
@@ -16,7 +16,7 @@ When you submit a job to a quantum processing unit (QPU), it enters the schedule
 *   **Group:** A mid-level structure to which access allocation can be allocated by the hub for one or more collections of users (projects).
 *   **Project:** The base-level construct to which time are allocated from the overarching group, and to which users are directly assigned.
 *   **Scheduling window:** The fair-share scheduler accounts for usage over a rolling time window. Only execution time accumulated within that window is accounted for the purpose of fairness. The length of that window is currently 28 days. When the fair-share scheduler is invoked, it takes into account usage starting 28 days ago.
-*   **Time used:** For every group and project, during the scheduling window, we account for all usage on all the QPUs of the IBM Quantum Premium Plan. These include all successful jobs, as well as jobs returning known select errors. It does not account for canceled jobs, even when partially executed.
+*   **Time used:** For every group and project, during the scheduling window, we account for all usage on all the QPUs of the IBM Quantum Premium Plan. These include all successful jobs, as well as jobs returning known select errors.
```
Comment (abbycross) [inline docs/guides/fair-share-scheduler.mdx:19]:
```suggestion
*   **Time used:** For every group and project, during the scheduling window, we account for all usage on all the QPUs of the IBM Quantum Premium Plan. These include all successful jobs, as well as jobs returning known select error codes.
```

**Pull request merged**